definitions/variable/technology/technologies.yaml

```diff
@@ -205,6 +205,13 @@
     description: Number of sold (new) vehicles with {Transport mode, tech and carrier}
     unit: million
 
+- Stock|Transportation|Charging Station:
+    description: Number of charging stations for electric vehicles
+    unit: million
+- Stock|Transportation|Charging Station|Public:
```

Comment:
In my opinion, this variable name is not sufficiently clear whether they are in public areas (how defined?) or owned by public entities (in contrast to those offered by private companies or on private/residential areas).

Also, please create a complete sub-categorization so that public/private/other charging stations  can be reported such that they be summed up to total number of charging stations.